docs/pages/9 - Contrib Modules.md

```diff
@@ -82,87 +82,101 @@ Quickstart:
   The package name of the object.
 
 
-### Other Mill Plugins
+## Thirdparty Mill Plugins
 
-- [ensime](https://github.com/yyadavalli/mill-ensime "mill-ensime")
-
-  Create an [.ensime](http://ensime.github.io/ "ensime") file for your build.
-  
-  Quickstart:
-  ```scala
-  import $ivy.`fun.valycorp::mill-ensime:0.0.1`
-  ```
-  ```sh
-  sh> mill fun.valycorp.mill.GenEnsime/ensimeConfig
-  ```
+### DGraph
   
-- [dgraph](https://github.com/ajrnz/mill-dgraph "mill-dgraph")
+Show transitive dependencies of your build in your browser.
 
-  Show transitive dependencies of your build in your browser.
-  
-  Quickstart:
-  ```scala
-  import $ivy.`com.github.ajrnz::mill-dgraph:0.2.0`
-  ```
-  ```sh
-  sh> mill plugin.dgraph.browseDeps(proj)()
-  ```
+Project home: https://github.com/ajrnz/mill-dgraph
```

Comment:
Should these be links?

Reply:
Of course, Those are supposed to be links, at least with typical Markdown renderers. I don't know how to check it, though.

Reply:
After comparing the current mill doc website with the source markdown, I'm convinced those links will also render as links.